UnitTest/TestScripts/commandTest.cpp

```diff
@@ -0,0 +1,79 @@
+/* commandTest.cpp tests the command module non-void functions
+ * Usage:
+ *  part of NS2 unit testing suite
+ *  to be called in unitTestDriver.cpp
+ *
+ */
+
+
+// NS2 includes
+#include "../headers/config.hpp"
+#include "../headers/commandHandling.hpp"
+
+/* - - - - - - testCheckMetaCommand - - - - - - *
+ * Usage:
+ * tests checkMetaCommand with all possible commands
+ * 
+ * Inputs:
+ *  none
+ *  
+ * Outputs:
+ *  0 - pass
+ *  1 - fail
+ */
+int testCheckMetaCommand() {

```

Comment:
Is this testing `executeCommand` instead of `checkMetaCommand`, I don't see how this test would pass?